Doc/library/time.rst

```diff
@@ -815,9 +815,21 @@ These constants are used as parameters for :func:`clock_getres` and
    .. versionadded:: 3.7
 
 
+.. data:: CLOCK_UPTIME_RAW
+
+   Clock that increments monotonically, tracking the time since an arbitrary 
+   point, unaffected by frequency or time adjustments and not increment while 
```

Comment:
(English is not my first language.) English question: is "not increment while..." correct? Or should it be "not incremented" as in "unaffected"?